### ISSUE TITLE
Save plot fails

### ISSUE THREAD

**Issue by conradek** [blink1073/oct2py#81]
Consider:

``` matlab
function [] = subplot_test

  clf;
  x = logspace (-5, 1, 10);
  y = logspace (-5, 1, 10);

  subplot (1,2,1);
  % semilogx (x, y);
  plot (x,y);
  xlabel ('semilogx (x, y)');

  subplot (1,2,2);
  % semilogx (-x, y);
  plot (x,y);
  xlabel ('semilogx (-x, y)');

  % saveas(gcf, "subplot_test.png");
  % print subplot_test.pdf;

  end;
```

I can run the above code successfully in either Octave or from this Python code:

``` python
from oct2py import octave, Oct2Py

oc = Oct2Py()
oc.eval("subplot_test()")
```

However, if I uncomment either the saveas or print command, oct2py throws an exception:

Traceback (most recent call last):
  File "subplot_test.py", line 4, in <module>
    oc.eval("subplot_test()")
  File "C:\Python27\Lib\site-packages\oct2py\core.py", line 284, in eval
    post_call=post_call)
  File "C:\Python27\Lib\site-packages\oct2py\core.py", line 807, in evaluate
    raise Oct2PyError(msg)
oct2py.utils.Oct2PyError: Oct2Py tried to run:
"""
subplot_test()
"""
Octave returned:
nth_element: n must be a scalar or a contiguous range

Running this in Octave, with either the saveas or print command uncommented executes as expected.

Any ideas on why this occurs?

I am running:

Windows 7 SP1
Octave 4.0.0
oct2py 3.3.3
Python 2.7.10

Thanks!

Conrad

**Comment by blink1073** [comment]
Hmm, I'm afraid I'm not sure what could be causing that.  I was able to create a plot file from your code, but I'm using OSX.  I'll take a look next time I log on to a Windows machine.

**Comment by conradek** [comment]
Glad you could generate a plot.  On OS X, could you also save a plot?

**Comment by blink1073** [comment]
Yes, I had to use an svg extension (quirk of the OSX version), but yes, I get an image.

**Comment by blink1073** [comment]
@conradek,  I just released 3.5 which should fix this, can you confirm?

**Comment by blink1073** [comment]
Closing due to lack of response, feel free to reopen.
